homeassistant/components/fortios/device_tracker.py

```diff
@@ -0,0 +1,111 @@
+"""
+Support to use FortiOS device like FortiGate as device tracker.
+
+This component is part of the device_tracker platform.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/fortiosapi/
+"""
+import logging
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.device_tracker import (
+    DOMAIN, PLATFORM_SCHEMA, DeviceScanner)
+from homeassistant.const import CONF_HOST, CONF_TOKEN
+from homeassistant.const import CONF_VERIFY_SSL
+
+REQUIREMENTS = ['fortiosapi==0.10.8']
+
+_LOGGER = logging.getLogger(__name__)
+DEFAULT_VERIFY_SSL = False
+
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Required(CONF_TOKEN): cv.string,
+    vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean
+})
+
+
+def get_scanner(hass, config):
+    """Validate the configuration and return a FortiOSDeviceScanner."""
+    from fortiosapi import FortiOSAPI
+
+    host = config[DOMAIN].get(CONF_HOST)
```

Comment:
I am not sure I understand what you want here. Is it the use of unnecessary local variables `host` that you want to prevent and just call `config[DOMAIN].get(CONF_HOST)` where it is needed?

Reply:
No. Just don't use `dict.get` when we don't need to do that. If we know that the key is in the dict, we should use `dict[key]`.

Reply:
OK. Thanks for the explanation, most appreciated! I'll fix that.